appendix-basic65.tex

```diff
@@ -1,8 +1,10 @@
 \addtocontents{toc}{\protect\setcounter{tocdepth}{1}}
 \hypersetup{bookmarksdepth=2}
 \chapter{BASIC 65 Command Reference}
+\label{cha:basic-reference}
 
-\section{Commands, Functions and Operators}
+
+\section{Commands, Functions, and Operators}
```

Comment:
It's called the Oxford comma and it's a matter of style. It is used in lists of three or more items. I use it because there are cases where it is needed to resolve ambiguity, and it's important to be consistent. This specific case is not necessarily ambiguous, but if there's a chance that it reads as two things and not three, it trips up the reader slightly trying to parse it, and I prefer to just always use it to keep the reader flowing forward.

https://www.grammarly.com/blog/what-is-the-oxford-comma-and-why-do-people-care-so-much-about-it/

Reply:
Ok, after reading that, I see where it's coming from now, though it still feels a little cringey and foreign to my eyes (a bit like the commas/full-stops before end-quote matter too). Ah well, will leave it be for now, maybe it'll grow on me. Though if it pops on the radar of other reviewers, maybe we can ponder it again.